pkg/oc/graph/appsgraph/edge_test.go

```diff
@@ -41,7 +41,7 @@ func TestNamespaceEdgeMatching(t *testing.T) {
 	fn("other", g)
 	AddAllDeploymentConfigsDeploymentEdges(g)
 
-	if len(g.Edges()) != 4 {
+	if len(g.Edges()) != 6 {
```

Comment:
Previously, two edges between the same nodes with opposite orientation were returned as one edge (even though this is a directed graph). Looks like this a result of an upstream fix.